hardware/ariadne/bootloaders/ariadne/main.c

```diff
@@ -133,6 +133,7 @@ int main(void)
 			}
 		}
 		/* Blink the notification led */
+		wdt_reset(); //Required so it doesn`t hang.
```

Comment:
Doesn't this make the wdt_reset() call at https://github.com/codebendercc/Ariadne-Bootloader/blob/master/hardware/ariadne/bootloaders/ariadne/main.c#L132 redundant?